build.sbt

```diff
@@ -12,7 +11,30 @@ inThisBuild(Seq[Setting[_]](
   developers := List(
     Developer("retronym", "Jason Zaugg", "@retronym", url("https://github.com/retronym")),
   ),
-  scalacOptions := Seq("-feature", "-deprecation", "-Xlint", "-Werror")
+  scalacOptions := Seq("-feature", "-deprecation", "-Xlint", "-Werror"),
+  githubWorkflowOSes := Seq("ubuntu-latest", "macos-latest", "windows-latest"),
+  githubWorkflowJavaVersions := Seq(
+    JavaSpec.temurin("8"),
+    JavaSpec.temurin("11"),
+    JavaSpec.temurin("17")
+  ),
+  githubWorkflowTargetTags ++= Seq ("v*"),
+  githubWorkflowPublishTargetBranches :=  Seq(
+    RefPredicate.StartsWith(Ref.Tag("v")),
+    RefPredicate.Equals(Ref.Branch("main"))
+  ),
+  githubWorkflowPublish := Seq (
+    WorkflowStep.Sbt(
+      commands = List("ci-release"),
+      name = Some("Publish project"),
+      env = Map(
+        "PGP_PASSPHRASE" -> "${{ secrets.PGP_PASSPHRASE }}",
```

Comment:
@SethTisue Is this where the credentials will sit? Its the standard place for sbt-ci-release.

Reply:
Yes.